### ISSUE TITLE
Warning about starvation MBean with sequential runs in SBT

### ISSUE THREAD

**Issue by hejfelix** [typelevel/cats-effect#3851]
When running a cats-effect app multiple times in the same shell, I get the warning below. Setting `fork := true` solves it, but I had to go to discord for this tip.

```scala
[WARNING] Failed to register Cats Effect CPU starvation MBean, proceeding with
no-operation versions. You will not see MBean metrics for CPU starvation.
Exception follows:
 javax.management.InstanceAlreadyExistsException: cats.effect.metrics:type=CpuStarvation
    at java.management/com.sun.jmx.mbeanserver.Repository.addMBean(Repository.java:436)
    at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerWithRepository(DefaultMBeanServerInterceptor.java:1865)
    at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerDynamicMBean(DefaultMBeanServerInterceptor.java:960)
    at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerObject(DefaultMBeanServerInterceptor.java:895)
    at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerMBean(DefaultMBeanServerInterceptor.java:320)
    at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.registerMBean(JmxMBeanServer.java:523)
    at main$ @ main.Main$.main(Main.scala:9)
    at main$ @ main.Main$.main(Main.scala:9)
    at main$ @ main.Main$.main(Main.scala:9)
    at main$ @ main.Main$.main(Main.scala:9)
    at main$ @ main.Main$.main(Main.scala:9)
```

**Suggestion**: Improve error message to hint that you might need `fork := true`

**Comment by kapunga** [comment]
Is this still an issue? I was looking for a good first issue to maybe take a crack at, but it looks like [this PR](https://github.com/typelevel/cats-effect/pull/3774) from @Daenyth was released in v3.5.2, the same day this issue was opened and it looks like it addressed this exact problem.

**Comment by armanbilge** [comment]
Thanks, I agree that https://github.com/typelevel/cats-effect/pull/3774 can close this issue!
